drivers/virtualbox/virtualbox.go

```diff
@@ -174,23 +179,71 @@ func (d *Driver) SetConfigFromFlags(flags drivers.DriverOptions) error {
 
 func (d *Driver) PreCreateCheck() error {
 	// Check that VBoxManage exists and works
-	if err := vbm(); err != nil {
-		return err
+	return vbm()
+}
+
+// IsVTXDisabled checks if VT-X is disabled in the bios. If it is, the vm will fail to start.
+// If we can't be sure it is disabled, we carry on and will check the vm logs after it's started.
+func (d *Driver) IsVTXDisabled() bool {
+	// TODO: Check on linux/OSX
+	if runtime.GOOS != "windows" {
+		return false
 	}
 
-	return nil
+	stdout, err := runCommand("wmic", "cpu", "get", "VirtualizationFirmwareEnabled")
+	if err != nil {
+		log.Debugf("Couldn't check that VT-X/AMD-v is enabled. Will check that the vm is properly created.")
+		return false
+	}
+
+	disabled := strings.Contains(string(stdout), "FALSE")
+	return disabled
 }
 
-func (d *Driver) Create() error {
-	if err := vbm("--version"); err != nil {
-		return fmt.Errorf("Checking VirtualBox version failed: %s", err)
+// runCommand runs a shell command.
+func runCommand(name string, args ...string) (string, error) {
```

Comment:
Since it actually calls `Output` in addition to running a command, why not name it `cmdOutput`?